doc/changes.rst

```diff
@@ -5,6 +5,12 @@ desitarget Change Log
 0.45.2 (unreleased)
 -------------------
 
+* Re-tune LRG cuts for DR9 and updating the LRG SV target bits [`PR #660`_].
```

Comment:
Unit tests are failing because you created a link to `PR #660` but didn't include the specific location to which to link. About a dozen lines down in the `changes.rst` file, you'll see:
```
.. _`PR #659`: https://github.com/desihub/desitarget/pull/659
```
Beneath this line, you need to add:
```
.. _`PR #660`: https://github.com/desihub/desitarget/pull/660
```

Reply:
Thanks. It's fixed. (I was wondering why the link did not work.)